examples/cx16/k_cbm_file_test.c

```diff
@@ -0,0 +1,282 @@
+// llvm-mos-sdk cx16 kernel file access tests
+//
+// vim: set et ts=4 sw=4
+
+#include <stddef.h>
```

Comment:
`k_cbm_file_test.c` does not seem to specific to x16, but would work on any commodore target. Could it be lifted to a `commodore/` folder? And what does `k_` signify in the cx16 tests?

Reply:
This is the prefix on the functions being exercised. I believe it was inherited from the cc65 "API" and stands for KERNAL [sic].  It _almost_ works on other CBM platforms, but not fully.  Also, I don't believe there is a "generic" commodore example folder (or any other CBM flavored examples - other than the ones I have added for X16).

Reply:
Thanks, I suspected it was `kernal` but it seems a little odd not knowing the history. I think it would be useful to add a `commodore/` folder and a test that works across all CBM targets. Perhaps this is for another PR, but we should probably avoid duplicate code. Could you add a comment pointing out where X16 specific stuff is?

Reply:
The main X16 issues I am aware of (e.g., from compiling with cc65) are mostly due to all the "normal ASCII" use all over.  Other issues are the X16 specific control characters used.  That said, I suspect pretty trivial to "port" (and perhaps make more ugly with 40 columns in all uppercase or add ASCII to PETSCII conversion [or maybe some exists - but I am not a fan of cc65 style "char mangling"]).  I'll add a few portability notes.